src/App/App.test.js

```diff
@@ -0,0 +1,163 @@
+
+  fetchTodaysColor.mockResolvedValue({color: 'mock color'});
+
+  getAllColors.mockResolvedValue({records: [
+    {
+      name: 'color one',
+      hex: '#123456',
+      id: 123456
+    },
+    {
+      name: 'color two',
+      hex: '#ABCDE',
+      id: 234546
+    },
+    {
+      name: 'color three',
+      hex: '#w38i7d',
+      id: 674843239
+    }
+  ]});
+
+  getArt.mockResolvedValue({ records: [
+    {
+      objectid: 12345,
+      primaryimageurl: 'url1.jpg',
+      title: 'piece1',
+      people: [{
+        displayname: 'artist1'
+      }]
+    },
+    {
+      objectid: 54321,
+      primaryimageurl: 'url2.png',
+      title: 'piece2',
+      people: [{
+        displayname: 'artist2'
+      }]
+    },
+    { 
+      objectid: 1082297,
+      title: 'title',
+      primaryimageurl: 'url.png',
+      people: [{
+        role: 'artist',
+        displayname: 'Leo D.V.'
+      }],
+      culture: 'culture',
+      century: 'century',
+      period: 'period',
+      medium: 'medium', 
+      technique: 'technique',
+      description: 'description',
+      images: [{
+        baseimageurl: 'url2.jpg'
+      }]
+    }
+  ]});
+
+  it('should render the loading message before mounting', () => {
+    const { getByText } = render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    );
+
+    const loading = getByText('Loading...');
+
+    expect(loading).toBeInTheDocument();
+  });
+  
+  it('should render correctly after mounting', async () => {
+    const { getByText } = render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    );
+
+    const todaysColor = await waitFor(() => getByText('today\'s color: mock color'));
+    const todaysGallery = await waitFor(() => getByText('view today\'s gallery'));
+    const otherColors = await waitFor(() => getByText('pick another color'));
+    const viewFavorites = await waitFor(() => getByText('view my gallery'));
+
+    expect(todaysColor).toBeInTheDocument();
+    expect(todaysGallery).toBeInTheDocument();
+    expect(otherColors).toBeInTheDocument();
+    expect(viewFavorites).toBeInTheDocument();
+  }); 
+
+  it('should be able to go to today\'s gallery', async () => {
+    const { getByText } = render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    );
+
+    const todaysGallery = await waitFor(() => getByText('view today\'s gallery'));
+    
+    fireEvent.click(todaysGallery);
+    
+    
+    const pieceOne = await waitFor(() => getByText('piece1'))
+    const artistOne = await waitFor(() => getByText('artist1'))
+    const pieceTwo = await waitFor(() => getByText('piece2'))
+    const artistTwo = await waitFor(() => getByText('artist2'));
+    
+    expect(pieceOne).toBeInTheDocument();
+    expect(artistOne).toBeInTheDocument();
+    expect(pieceTwo).toBeInTheDocument();
+    expect(artistTwo).toBeInTheDocument();
+  });
+
+  it.skip('should be able to go to a specific art page from today\'s gallery', async () => {
```

Comment:
Getting errors on this test (and another) that say: ` Unable to find the "window" object for the given node. Please file an issue with the code that's causing you to see this error: https://github.com/testing-library/dom-testing-library/issues/new`
I am unsure of how to deal with this.